core/src/main/java/com/radixdlt/rev2/modules/REv2StateManagerModule.java

```diff
@@ -188,7 +191,8 @@ public static REv2StateManagerModule createForTesting(
         ledgerSyncLimitsConfig,
         protocolConfig,
         noFees,
-        scenariosExecutionConfig);
+        scenariosExecutionConfig,
+        false);
```

Comment:
Why does a testing state manager module not install persistence? This is surprising to me?

Reply:
It was a surprise to me too, but it appeared that `FunctionalRadixNodeModule` unconditionally installs mocked `AddressBookPersistence` and this clashes with the persistence configured here.